snippets/view.cson

```diff
@@ -0,0 +1,102 @@
+'.source.coffee':
+
+
+  'available in view':
+    'prefix': 'view--help'
+    'body':
+      """
+      ${1:
+        # you can use these snippets within a
+        # Em.view.extend(
+        #   example : tN<tab>
+        #   _____________________________________________
+        #  | type or long version ->  press tab to see:  |
+        #  |----- -> -----------  ->  ------------------ |
+        #  | tN   ->  tagN        ->  tagNames:          |
+        #  | aB   ->  attributeB  ->  attributeBindings: |
+        #  | cN   ->  classN      ->  classNames:        |
+        #  | cNB  ->  classNB     ->  classNameBindings: |
+        #  | eM   ->  eventM      ->  eventManager:      |
+        #  |_____________________________________________|
+        #
+        #   *press <Backspace> to remove this quick guide.
+
+            }
+      """
+
+  '0 Ember View':
+    'prefix': 'view'
+    'body':
+      """
+        _export = Ember.View.extend(
+          ${1:# to see more available snippets by Expanding :
+
+          # view-help${2}  (or view-h)
+          }
+        )
+      """
+
+  '1 Ember event Manager':
+    'prefix': 'eventM'
```

Comment:
Can you rename this and following prefixes to long version `eventManager` instead of `eventM`, same for others too